addons/account/models/company.py

```diff
@@ -176,6 +176,10 @@ class ResCompany(models.Model):
     # Storno Accounting
     account_storno = fields.Boolean(string="Storno accounting", readonly=False)
 
+    # Multivat
+    fiscal_position_ids = fields.One2many(comodel_name="account.fiscal.position", inverse_name="company_id")
+    is_multi_vat_company = fields.Boolean(string="Is the company submitting tax report in other countries", compute="_compute_is_multi_vat_company", store=True)
```

Comment:
I find it a bit odd that the same `read_group()` is used here just to set a boolean, and on `account.tax.repartition.line` to retrieve the country ids, it's doing the same thing twice :thinking: 

Why not store the country ids directly on the company in a variable like `multi_vat_country_ids`, and use it directly in `_compute_tag_ids_domain()` to build the domain, saving the `read_group()` there ?